frontend/coprs_frontend/coprs/templates/coprs/detail/settings/integrations.html

```diff
@@ -67,6 +67,42 @@ <h3> Custom webhook(s) </h3>
         <div class="well well-sm">
             {{ custom_dir_url }}
         </div>
+
+        <h3>Webhook History</h3>
+        <div>
+            {% if webhook_history %}
+            <table class="datatable dataTable table table-striped table-bordered">
+                <thead>
+                  <tr>
+                    <th>Timstamp</th>
+                    <th>Webhook ID</th>
+                    <th>User Agent</th>
+                  </tr>
+                </thead>
+                {% for webhook in webhook_history %}
+                <tr>
+                    <td  class="webhook_timestamp" data-toggle="tooltip" title="{{webhook.timestamp}}">{{webhook.timestamp|humanize_datetime()}}</td>
```

Comment:
Looking at some dates that we already print on the page, e.g.
https://copr.fedorainfracloud.org/coprs/g/copr/copr/build/7470434/

The date format is `2024-05-21 12:49 UTC (3 months ago)`. I'd like to have the same format also here. Please see my comment above regarding the `localized_time` filter.

Then the `title` could be the UNIX timestamp?